testing/README.md

```diff
@@ -390,9 +393,55 @@ Visual regression testing allows you to be confident that changes made to the co
 [↑ scrollTo('#table-of-contents')](#table-of-contents)
 
 
-
 ## React
 
 If your project is using React, you can find React-specific testing recommendations in the [React styleguide](https://github.com/Shopify/javascript/tree/master/react#testing).
 
 [↑ scrollTo('#table-of-contents')](#table-of-contents)
+
+## Coverage
+
+Generating a test coverage report can help you identify areas of the code that are lacking tests. Coverage reports can be integrated with CodeCov and Circle CI to ensure sufficient coverage with every checkin. See the [CodeCov Integration Guide](https://vault.shopify.com/Codecov-integration) for more information.
```

Comment:
might be worth mentioning that we’re not striving for 100% coverage usually. as you say, generating coverage reports should help us find untested code, but usually we don’t set a hard number like “we must have 90% coverage”.

Reply:
Some teams do have a strict coverage threshold, I think it's fair for that to be decided on a team-by-team basis.

Reply:
How about: 
> Generating a test coverage report can help you identify untested code. Coverage reports can be integrated with CodeCov and Circle CI to ensure team-specific coverage targets are met with every checkin.